worlds/musedash/__init__.py

```diff
@@ -63,6 +63,11 @@ class MuseDashWorld(World):
 
     item_name_to_id = {name: code for name, code in md_collection.item_names_to_id.items()}
     location_name_to_id = {name: code for name, code in md_collection.location_names_to_id.items()}
+    item_name_groups = {
+        "Songs": {name for name in md_collection.song_items.keys()},
+        "FillerItems": {name for name in md_collection.filler_items.keys()},
```

Comment:
It's up to you, but you might want to use a more user-friendly name
```suggestion
        "Filler Items": {name for name in md_collection.filler_items.keys()},
```